docs/jvs-apis.md

```diff
@@ -0,0 +1,107 @@
+# JVS Usage
```

Comment:
Let's **not** mix API doc vs. quick setup doc.

API doc - focusing on the API spec. How to call the API and what's the response, etc.
Setup doc - how to setup JVS in a cloud project and sample interactions

Feel free to separate them in two PRs (I haven't added the lumberjack setup doc).

Reply:
done